draft-ietf-rats-corim.md

```diff
@@ -1000,6 +1000,41 @@ measurements for the Target Environment.
 
 [^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/10
 
+# CoBOM {#sec-cobom}
+
+A BOM object represents the signal for the verifier to activate the listed
+tags. Data contained in a tag MUST NOT be used for appraisal until a BOM which
+activates that tag has been received and successfully processed. All the tags
+listed in the BOM must be activated in the same transaction, i.e., either all
+or none.
```

Comment:
There should be normative language surrounding the scope of the activation. Possibly: All tags listed in `tags-list` MUST be activated or no tags are activated.

Reply:
done in a5638b2